changelog/8.17.9.asciidoc

```diff
@@ -0,0 +1,35 @@
+// begin 8.17.9 relnotes
+
+[[release-notes-8.17.9]]
+==  8.17.9
+
+Review important information about the  8.17.9 release.
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
```suggestion

```

nit: a bunch of useless whitespace, also the changelog entries capitalize everyword unnecessarily